cmd/zar/README.md

```diff
@@ -0,0 +1,347 @@
+# zar overview
+
+> DISCLAIMER: ZAR IS A CURRENTLY A PROTOTYPE UNDER DEVELOPMENT AND IS
+> CHANGING QUICKLY.  PLEASE EXPERIMENT WITH IT AND GIVE US FEEDBACK BUT
+> IT'S NOT QUITE READY FOR PRODUCTION USE. THE SYNTAX/OPTIONS/OUTPUT ETC
+> ARE ALL SUBJECT TO CHANGE.
+
+This is a sketch of an early prototype of zar and related tools for
+indexing and search log archives.
+
+We'll use the test data here:
+```
+https://github.com/brimsec/zq-sample-data/tree/master/zng
+```
+You can copy the just zng data directory needed for this demo
+into your current directory using subversion:
+```
+svn checkout https://github.com/brimsec/zq-sample-data/trunk/zng
+```
+Or, you can clone the whole data repo using git and symlink the zng dir:
+```
+git clone https://github.com/brimsec/zq-sample-data.git
+ln -s zq-sample-data/zng
+```
+
+## ingesting the data
+
+Let's take those logs and ingest them into a directory.   We'll make it
+easier to run all the commands by setting an environment variable pointing
+to the root of the logs tree.
+```
+mkdir ./logs
+setenv ZAR_ROOT `pwd`/logs
```

Comment:
Is there any hope of changing this to:

```
export ZAR_ROOT=`pwd`/logs
```

...to please the Bourne-derived masses rather than the `csh` fetishists? :)

Reply:
I'll change it to "set", which works across shells.

Reply:
Someday I'll switch to bash :-)